doc/usage.rst

```diff
@@ -458,10 +459,15 @@ use :meth:`pooch.Pooch.load_registry` to read them:
         # We'll load it from a file later
         registry=None,
     )
-    GOODBOY.load_registry(os.path.join(os.path.dirname(__file__), "registry.txt"))
-
-The ``registry.txt`` file in this case is in the same directory as the ``datasets.py``
-module and should be shipped with the package. It's contents are:
+    # Get registry file from package_data
+    registry_file = pkg_resources.resource_stream("plumbus", "registry.txt")
+    # Load this registry file
+    GOODBOY.load_registry(registry_file)
+
+The ``registry.txt`` file in this case is in the ``plumbus/`` directory module and should be
+shipped with the package. We use `pkg_resources <https://setuptools.readthedocs.io/en/latest/pkg_resources.html#basic-resource-access)`__
+to access it, giving it the name of our Python package (the one we'll put in ``setup.py``).
+Its contents are:
```

Comment:
I wasn't sure what width is used for word-wrapping rst files. Also this link is too long :sweat_smile:

Reply:
:smile: for rst files it doesn't really matter too much. I usually do 79 or 88 (whichever you prefer). Not much we can do about the link, though.